control-plane/agents/src/bin/core/nexus/service.rs

```diff
@@ -180,4 +194,13 @@ impl Service {
         let mut nexus = self.specs().nexus_opt(&request.nexus).await?;
         nexus.as_mut().remove_child(&self.registry, request).await
     }
+
+    /// Gets rebuild history details for a nexus..
```

Comment:
`Gets rebuild history details for a nexus..` --> `Gets rebuild history details for a nexus.`